provider/clusterrosa/hcp/resource.go

```diff
@@ -364,6 +364,10 @@ func (r *ClusterRosaHcpResource) Schema(ctx context.Context, req resource.Schema
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"worker_disk_size": schema.Int64Attribute{
```

Comment:
if this one cannot be updated after cluster creation, should check it in validateNoImmutableAttChange() func